src/index.js

```diff
@@ -4,6 +4,4 @@ const dt2js = require('./dt2js')
 const js2dt = require('./js2dt')
 
 module.exports.dt2js = dt2js.dt2js
-module.exports.dt2js.setBasePath = dt2js.setBasePath
```

Comment:
> Is there any situation where setting a different path may be needed though

As old README said:
> when the inputed RAML contains `!include`s and those includes are not in the same directory as the script it is being ran from, you can use `dt2js.setBasePath()` to specify a different base path.

Reply:
But now webapi-parser defaults to the path of the root RAML file, right? If so, do you think there may be a valid situation requiring to override the base path in which case we may want to consider re-adding this method... thoughts?

Reply:
I've just tested and it looks like webapi-parser and amf resolve references relative to a script location when parsing a string and relative to a root file when parsing a file path.
Note that both old and new versions of dt2js accept RAML string as input instead of file path.

So here's how the new version of dt2js behaves:
When converting type from a RAML string, references are resolved relative to the location of a script which performs parsing. Also references have to be `!include file:///something` instead of `!include /something`.

So it looks like the idea behind `setBasePath` would be useful in the new version as well.

Few ways we could fix this:
1. Accept file path in dt2js instead of  RAML string. But this will limit dt2js usage and change the old API.
2. Implement an idea of `setBasePath`. AMF parsers support second string argument when parsing a RAML string - base path to use. webapi-parser doesn't support that argument but we could add its support.

Reply:
> 1. Accept file path in dt2js instead of RAML string

The command-line interface does that already, right? Are you saying that the new `webapi-parser` version doesn't do that or are you suggesting the JS/code interface takes file paths/URLs INSTEAD of a string?

I am ok with re-implementing `setBasePath`. Will it support remote http URLs?

Reply:
Command-line version takes file paths and js code version takes RAML/JSON content strings - both old and new versions behave like this.
I'm suggesting passing file paths instead of content strings to js code version as one of possible solutions.

> I am ok with re-implementing setBasePath. Will it support remote http URLs?

Depends on whether AMF supports that. I'm not sure if it does.

Reply:
> Command-line version takes file paths
Makes sense for a CLI. Let's keep it this way.

> js code version takes RAML/JSON content strings
makes sense here too. I think we should keep this behavior. Especially if it's the same behavior as `AMF`/`webapi-parser`. We should keep the behaviors as close as possible.

> I'm not sure if it does
it does

So let's add `setBasePath()` then? I'm also happy with adding a 2rd optional argument to the `dt2js()` and `js2dt()` methods if that fits better with how `webapi-parser` works.

Reply:
> So let's add setBasePath() then? I'm also happy with adding a 2rd optional argument to the dt2js() and js2dt() methods if that fits better with how webapi-parser works.

Ok, I've already created gh issue in webapi-parser for it https://github.com/raml-org/webapi-parser/issues/35